test/sth_app_test_helper.js

```diff
@@ -0,0 +1,149 @@
+/* globals module */
+
+(function() {
+  "use strict";
+
+  var sthTestConfiguration;
+  var sthConfiguration;
+  var sthDatabase;
+  var sthHelper;
+
+  function randomDate(start, end) {
```

Comment:
JSDoc comments included! ;) Sorry about that, I forgot to do it from the beginning ;) I'll mention you as soon as I pushed the changes to the repo ;) Thanks!